php/cloudbuild.yaml

```diff
@@ -18,32 +18,44 @@ options:
 steps:
 
 # PHP 8.0
-- name: gcr.io/cloud-builders/docker
+- id: 'build-php80'
+  name: gcr.io/cloud-builders/docker
   args: ['build', '-t', 'gcr.io/cloud-devrel-kokoro-resources/php80', '.']
   dir: 'php/php80'
   waitFor: ['-']
 
 # PHP 8.1
-- name: gcr.io/cloud-builders/docker
+- id: 'build-php81'
+  name: gcr.io/cloud-builders/docker
   args: ['build', '-t', 'gcr.io/cloud-devrel-kokoro-resources/php81', '.']
   dir: 'php/php81'
   waitFor: ['-']
 
 # PHP 8.2
-- name: gcr.io/cloud-builders/docker
+- id: 'build-php82'
+  name: gcr.io/cloud-builders/docker
   args: ['build', '-t', 'gcr.io/cloud-devrel-kokoro-resources/php82', '.']
   dir: 'php/php82'
   waitFor: ['-']
 
-# PHP 7.3 / 7.4 / 8.0
-# (do not build this for now, as it is not in use)
-#- name: gcr.io/cloud-builders/docker
-#  args: ['build', '-t', 'gcr.io/cloud-devrel-kokoro-resources/php', '.']
-#  dir: 'php/php'
-#  waitFor: ['-']
+# Tag alternate names
+- name: gcr.io/cloud-builders/docker
+  args: ['tag', 'gcr.io/cloud-devrel-kokoro-resources/php80',
+                'gcr.io/cloud-devrel-public-resources/php80']
+  waitFor: ['build-php80']
+- name: gcr.io/cloud-builders/docker
+  args: ['tag', 'gcr.io/cloud-devrel-kokoro-resources/php81',
+                'gcr.io/cloud-devrel-public-resources/php81']
+  waitFor: ['build-php81']
+- name: gcr.io/cloud-builders/docker
+  args: ['tag', 'gcr.io/cloud-devrel-kokoro-resources/php82',
+                'gcr.io/cloud-devrel-public-resources/php82']
+  waitFor: ['build-php82']
```

Comment:
suggestion: you can tag multiple times on the docker build command as well:

`docker build -t tag1 -t tag2 .`

Reply:
Will be addressed by https://github.com/googleapis/testing-infra-docker/pull/372